### ISSUE TITLE
Latest Version 0.16.0 Has Broken Urlib/Boto3

### ISSUE THREAD

**Issue by jimmycfa** [aws-samples/sagemaker-run-notebook#7]
The latest version of boto3 and therefore latest container built from this has the same issue listed [here](https://github.com/boto/botocore/issues/2187 ) Consider adding urllib3==1.25.10 to the Dockerfile.

**Comment by jimmycfa** [comment]
My workaround was to add `urllib3==1.25.10` to a requirements.txt file and run: `run-notebook create-container --requirements requirements.txt`

**Comment by tomfaulhaber** [comment]
@jimmycfa Are you sure that you're running v0.16.0? This should have been fixed in 07f4ce0. Testing locally, I'm able to use `run-notebook build-container` to build a container that I can then run (either directly in the container or from a notebook execution run), the following code:

```python
import boto3
sess = boto3.Session()
sm = sess.client("sagemaker")
sm.list_processing_jobs(MaxResults=5)
```

works without problems.

Can you perform the following?

1. confirm that you're really running v0.16.0 by running `run-notebook --version`
2. rebuild the container without your workaround, 
3. run a notebook execution with the above code

If this fails, can you post the output of the container build process and the error you're seeing when you run the notebook?

Thanks!

**Comment by tomfaulhaber** [comment]
No worries. I had the same problem myself which is how I knew to fix it. Enjoy!
